documentation/ingestion_server/guides/mapping.md

```diff
@@ -0,0 +1,60 @@
+# Mapping database tables to Elasticsearch
+
+In order to synchronize a given table to Elasticsearch, the following
+requirements must be met:
+
+- The database table must have an autoincrementing integer primary key named
+  `id`.
+- A SyncableDoctype must be defined in `es_syncer/elasticsearch_models`. The
```

Comment:
`es_syncer` seems to be a holdover name of some sort, it doesn't appear anywhere in the project's directories. Can you update these references to `ingestion_server/...`?